### DISCUSSION TITLE
Scraping comments and reacts without getting blocked

### DISCUSSION THREAD

**Discussion by caribmorn** [kevinzg/facebook-scraper#823] [General]
Hi all,

Checked through quite a few pages of stuff and couldn't find people with a similar experience -

I am trying to scrape react(or)s and comment(er)s of all posts of a group.  

The thing is, my accounts keep getting blocked after about 50 posts.. I understand that reactors and commenters are a kind of "heavy" request as far as getting blocked quickly, but I was wondering if anyone has any tips (or comments whether this is normal behaviour on facebook's part to ban me after 50 posts)?  I see people saying you should be able to get thousands of posts, is that only without commenters and reactors?

I've tried this with new facebook accounts and also aged accounts with hundreds of friends.

Any tips about how to avoid this, or if it's unavoidable, how long to wait to continue the scrape is appreciated.

Thanks for any info!

**Comment by neon-ninja** [comment]
>whether this is normal behaviour on facebook's part to ban me after 50 posts

It depends how many reactors and comments you're extracting. If it's in the order of thousands per post, then absolutely, yes.

>you should be able to get thousands of posts, is that only without commenters and reactors?

Yes

>Any tips about how to avoid this

Scrape slower or less

**Reply by iankoratskydcn** [reply]
Is there any way to introduce a wait time between requests?
